### PR TITLE
Issue #3 First Attempt Complete

### PR DESCRIPTION
**Pull request by robecampbelljr**
[Issue #3 ](https://github.com/Vets-Who-Code/js-web-path-robecampbelljr/issues/3)
[Netlify Preview](https://friendly-caramel-5abc1e.netlify.app/) available.
I chose to make this about basic goal setting. The form will allow the user to add a new goals to the goals table along with a checklist where you can add up to three list items as steps for your goal.

### CODE REVIEW

**Comment by robecampbelljr** [comment]
> https://docs.netlify.com/site-deploys/deploy-previews/

I attempted to follow the instructions on [Netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) provided in the [issue #3](https://github.com/Vets-Who-Code/js-web-path-robecampbelljr/issues/3) References section. When I attempted to connect Netlify to the repository, [says it must be done by an organization owner.](https://drive.google.com/file/d/1lxwwEPaOfczOtyK3vMawnY1h4UNR_BMO/view?usp=sharing)

**Comment by axecopfire** [comment]
Interesting, it's enabled for all repos in the org...

**Comment by axecopfire** [comment]
test deploy preview

File: issue-3/about.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  </head>
+  <body>
+    <script src="./lib.js"></script>
+    <div class="app">
+      <header>
+        <nav>
+          <ul> Menu
+            <a href="./index.html"><li>Home</li></a>
+            <a href="./about.html"><li>About</li></a>
+            <a href="#"><li>Contact</li></a>
```
Comment (axecopfire) [inline issue-3/about.html:19]: `li` should wrap `a`

File: issue-3/about.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="utf-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="./style.css" />
+    <script>
+      document.documentElement.className = localStorage.getItem('theme');
+    </script>
+  </head>
+  <body>
+    <script src="./lib.js"></script>
+    <div class="app">
```
Comment (axecopfire) [inline issue-3/about.html:13]: why is this `div` here?

File: issue-3/index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          </script>
+        </label>
+      </header>
+      <section class="first">
+        <script>setFirstSection()</script>
+        <p>Goal setting is the process of defining targets or objectives that one intends to achieve within a specific time frame. <strong>Setting SMART goals</strong>, which are specific, measurable, achievable, relevant, and time-bound, can increase the chances of success. A good starting point is to identify what you want to achieve and why, and then break down the goal into smaller, more manageable steps. <strong>Visualizing success</strong> and regularly reviewing progress can help to stay motivated and focused. Setting goals can lead to a sense of direction and purpose, and <a href="https://en.wikipedia.org/wiki/Motivation">motivate</a> people to work towards their aspirations. It can also promote personal growth and development, improve performance, and increase satisfaction and happiness.</p>
+        <p>Use the form below to add another goal to the goal list. All fileds will <em>only accept</em> alpha numeric characters with the exception of the description field; it will accept basic punctuation.</p>
+      </section>
+      <hr>
```
Comment (axecopfire) [inline issue-3/index.html:38]: `<hr />` instead of `<hr>`

File: issue-3/index.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          </tbody>
+          <tfoot>
+
+          </tfoot>
+        </table>
+        <script>
+          setTable();
+        </script>
+      </section>
+      <br>
```
Comment (axecopfire) [inline issue-3/index.html:103]: `<br />` instead of `<br>`

**Review by axecopfire — APPROVED** [review]

**Pull request merged**